fairer-iOS/fairer-iOS/Screens/OnboardingProfile/UIComponent/OnboardingProfileGroupCollectionView.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  HomeGroupCollectionView.swift
+//  fairer-iOS
+//
+//  Created by Mingwan Choi on 2022/09/17.
+//
+
+import UIKit
+
+import SnapKit
+
+protocol SelectedProfileImageViewDelegate: AnyObject {
+    func showSelectedProfileImage(image: UIImage)
+}
+
+final class OnboardingProfileGroupCollectionView: BaseUIView {
+    let profileList: [UIImage] = [ImageLiterals.profileBlue3, ImageLiterals.profileBlue4, ImageLiterals.profileOrange1, ImageLiterals.profilePink1, ImageLiterals.profilePink3, ImageLiterals.profileOrange2, ImageLiterals.profileYellow2, ImageLiterals.profileIndigo3, ImageLiterals.profilePurple1, ImageLiterals.profilePurple2, ImageLiterals.profilePurple3, ImageLiterals.profileGreen1, ImageLiterals.profileYellow1, ImageLiterals.profileGreen3, ImageLiterals.profileLightBlue1, ImageLiterals.profileLightBlue2]
+    
+    private enum Size {
+        static let collectionHorizontalSpacing: CGFloat = 33
+        static let collectionVerticalSpacing: CGFloat = 16
+        static let cellSize: CGFloat = UIScreen.main.bounds.width * 0.15
+        static let collectionInsets = UIEdgeInsets(
+            top: collectionVerticalSpacing,
+            left: collectionHorizontalSpacing,
+            bottom: collectionVerticalSpacing,
+            right: collectionHorizontalSpacing)
+    }
+    weak var delegate: SelectedProfileImageViewDelegate?
+    private lazy var selectedProfileName = ImageLiterals.profileNone
+
+    // MARK: - property
+    
+    private let collectionViewFlowLayout: UICollectionViewFlowLayout = {
+        let flowLayout = UICollectionViewFlowLayout()
+        flowLayout.scrollDirection = .vertical
+        flowLayout.sectionInset = Size.collectionInsets
+        flowLayout.itemSize = CGSize(width: Size.cellSize, height: Size.cellSize)
+        flowLayout.minimumLineSpacing = 24
+        return flowLayout
+    }()
+    private lazy var collectionView: UICollectionView = {
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: collectionViewFlowLayout)
+        collectionView.backgroundColor = .clear
+        collectionView.dataSource = self
+        collectionView.delegate = self
+        collectionView.showsVerticalScrollIndicator = false
+        collectionView.register(cell: OnboardingProfileGroupCollectionViewCell.self,
+                                forCellWithReuseIdentifier: OnboardingProfileGroupCollectionViewCell.className)
+        return collectionView
+    }()
+    
+    // MARK: - life cycle
+    
+    override func render() {
+        self.addSubview(collectionView)
+        collectionView.snp.makeConstraints {
+            $0.leading.trailing.top.bottom.equalToSuperview()
```

Comment:
p3;

```suggestion
            $0.edges.equalToSuperview()
```
로 쓸 수 있습니당 ^______^